src/test/java/com/chat/yourway/integration/service/impl/ContactServiceImplTest.java

```diff
@@ -1,64 +1,82 @@
 
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-
 import com.chat.yourway.dto.request.ChangePasswordDto;
 import com.chat.yourway.exception.NoEqualsPasswordException;
+import com.chat.yourway.integration.extension.PostgresExtension;
+import com.chat.yourway.integration.extension.RedisExtension;
 import com.chat.yourway.model.Contact;
 import com.chat.yourway.repository.ContactRepository;
 import com.chat.yourway.service.ContactService;
-import org.junit.jupiter.api.BeforeEach;
+import com.github.springtestdbunit.annotation.DatabaseSetup;
+import com.github.springtestdbunit.DbUnitTestExecutionListener;
+import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.security.crypto.password.PasswordEncoder;
-import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.support.DependencyInjectionTestExecutionListener;
+import org.springframework.test.context.support.DirtiesContextTestExecutionListener;
+import org.springframework.test.context.transaction.TransactionalTestExecutionListener;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
 
+@ExtendWith(PostgresExtension.class)
+@ExtendWith(RedisExtension.class)
 @SpringBootTest
-@ActiveProfiles("test")
-@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@TestExecutionListeners(value = {
+        TransactionalTestExecutionListener.class,
+        DirtiesContextTestExecutionListener.class,
+        DependencyInjectionTestExecutionListener.class,
+        DbUnitTestExecutionListener.class
+})
 public class ContactServiceImplTest {
-    private Contact contact;
+    private static final String USERNAME = "username12345";
+
     @Autowired
     private ContactService contactService;
-
     @Autowired
     private PasswordEncoder passwordEncoder;
-
     @Autowired
     private ContactRepository contactRepository;
 
-    @BeforeEach
-    public void setUp() {
-        contact = Contact.builder()
-                .username("username12353")
-                .email("user@gmail.com")
-                .password(passwordEncoder.encode("oldPassword"))
-                .isActive(true)
-                .isPrivate(true)
-                .build();
-    }
-
+    @DisplayName("should change password when user passed correct password")
     @Test
-    public void testChangePassword_Success() {
-        ChangePasswordDto request = new ChangePasswordDto("oldPassword", "newPassword");
+    @DatabaseSetup("/dataset/contacts.xml")
```

Comment:
please, specify operation - INSERT|
also you need to use @DatabaseTeardown to delete entities created for specific test